lib/internal/http2/core.js

```diff
@@ -1720,6 +1721,10 @@ class Http2Stream extends Duplex {
     return !!(this[kState].flags & STREAM_FLAGS_HEADERS_SENT);
   }
 
+  get bytesWritten() {
+    return this[kState].bytesWritten;
```

Comment:
Can you avoid tracking this manually and instead return `this[kHandle].bytesWritten` directly?

Reply:
(And, ideally, expose `bytesRead` in the same go)